GTG/gtk/browser/browser.py

```diff
@@ -420,8 +418,63 @@ def _init_accelerators(self):
                                        Gtk.AccelFlags.VISIBLE)
 
 # HELPER FUNCTIONS ##########################################################
-    def temporary_search(self, widget):
-        self.quickadd_entry.grab_focus()
+
+    def on_search_toggled(self, widget):
+        search = self.builder.get_object("search_button")
```

Comment:
Do you mind to get those references directly in `__init__()` and reference them afterwards as `self.search_button` and `self.searchbar`?